apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -197,29 +190,6 @@ export const editSectionLoginType = (sectionId, loginType) => dispatch => {
   return dispatch(finishEditingSection());
 };
 
-export const updateShareSetting = (sectionId, shareSetting) => dispatch => {
-  dispatch({type: UPDATE_SHARING_REQUEST});
-  return new Promise((resolve, reject) => {
-    $.ajax({
-      url: `dashboardapi/sections/${sectionId}/update_sharing_disabled`,
```

Comment:
I left this API in because I'm not sure if we'll want to keep this functionality around for any reason.  AFAIK it's not being used now, so I could remove in a follow up PR?

Reply:
Good call, let's leave it for now. It's possible we want to use it later or that something external relies on it?